CHANGELOG.md

```diff
@@ -31,6 +31,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Blurred icons in KDE task switcher (alacritty.ico is now high-res)
 - Consecutive builds failing on macOS due to preexisting `/Application` symlink
 
+### Removed
+
+- `auto_scroll` config option as well as behavior
```

Comment:
I'm not sure why, but "behavior" sounds a bit strange here. I don't have a great suggestion for improvement, but maybe using what you've already used in the removal warning can make this a little nicer to read?

```suggestion
- Config option `auto_scroll` and its corresponding feature
```

Reply:
since your suggestion mirrors the thing in warning it makes sense.